pkgs/development/python-modules/pylink-square/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, buildPythonPackage, python3Packages, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "pylink-square";
+  version = "0.8.1";
+
+  src = fetchPypi {
+   inherit pname version;
+   sha256 = "1r062scm1dk58ji2329h2yf3iz18br0scn29d48w58icbzipr6wq";
+  };
+
+  doCheck = false;
```

Comment:
The upstream project has [tests](https://github.com/square/pylink/tree/master/tests). If GitHub is used as source then it would be perhaps possible to run them.

Reply:
Fixed.